src/OrchardCore.Modules/OrchardCore.Contents/Scripting/ContentMethodsProvider.cs

```diff
@@ -36,7 +36,14 @@ public ContentMethodsProvider()
                 var contentManager = serviceProvider.GetRequiredService<IContentManager>();
                 var contentItem = contentManager.NewAsync(contentType).GetAwaiter().GetResult();
                 contentItem.Merge(properties);
-                var result = contentManager.UpdateValidateAndCreateAsync(contentItem, publish == true ? VersionOptions.Published : VersionOptions.Draft).GetAwaiter().GetResult();
+                contentManager.UpdateAsync(contentItem).GetAwaiter().GetResult();
```

Comment:
Why is it necessary to update in this scripting 'createContentItem' method; shouldn't this behave the same now?

Reply:
I don't really know the answer to this. Before this PR, it was doing the following actions 

1. Update
2. Validate
3. Create 

Now, since it's a new contentItem, we only Validate then Create without calling the Update.

Reply:
It looks suspicious to me when we are creating a content item differently in this case. I think this should be the same everywhere.

Reply:
@gvkries my mistake. What I described above and what the code was doing was not the same. I explained what should happen. But the code was not doing the same thing. I updated the code to not call Update, validate and create only. Please let me know if you see anything else.